src/signals/incident-management/containers/IncidentDetail/index.js

```diff
@@ -171,12 +160,7 @@ export class IncidentDetail extends React.Component {
               {incident && (
                 <Row>
                   <Column span={12}>
-                    <DetailHeader
-                      incidentId={incident.id}
-                      status={incident?.status?.state}
-                      links={incident?._links}
-                      onPatchIncident={onPatchIncident}
-                    />
+                    <DetailHeader incident={incident} baseUrl="/manage" onPatchIncident={onPatchIncident} />
```

Comment:
This change reverts a change that I applied in a previous PR. This won't work.

Reply:
Good point. Changed.